src/browser/atom-window.coffee

```diff
@@ -73,6 +73,9 @@ class AtomWindow
     @browserWindow.loadUrl @getUrl(loadSettings)
     @browserWindow.focusOnWebView() if @isSpec
 
+    if locationsToOpen.length is 1 and locationsToOpen[0].pathToOpen is null
```

Comment:
Dislike this. Maybe a better option would be to pass an empty `locationsToOpen` to the constructor, or an option `openInitialEmptyBuffer`, then build this `locationsToOpen` hash.